wandb/sdk/internal/sender.py

```diff
@@ -1209,6 +1216,15 @@ def send_link_artifact(self, record: "Record") -> None:
             except Exception as e:
                 logger.warning("Failed to link artifact to portfolio: %s", e)
 
+    def send_use_artifact(self, record: "Record") -> None:
+        """
+        This function doesn't actually send anything, it is just used
+        internally
+        """
+        use = record.use_artifact
+        if use.type == "job":
+            self._job_builder.set_used_job(True)
```

Comment:
seems to be missing coverage here?

Reply:
There are several tests that cover this, for example all of the end to end functional tests and the test in `test_launch_run.py`. Adding an another test for coverage here doesn't seem like it will provide value, IMO

Reply:
ok, if you trust this covered i'm fine, just saw the missing coverage